[2024-12-19T08:54:53.747+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-19T08:54:53.790+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.first_task scheduled__2023-01-01T00:00:00+00:00 [queued]>
[2024-12-19T08:54:53.822+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.first_task scheduled__2023-01-01T00:00:00+00:00 [queued]>
[2024-12-19T08:54:53.823+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-19T08:54:53.918+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): first_task> on 2023-01-01 00:00:00+00:00
[2024-12-19T08:54:53.933+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=6229) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-19T08:54:53.935+0000] {standard_task_runner.py:72} INFO - Started process 6244 to run task
[2024-12-19T08:54:53.940+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'first_task', 'scheduled__2023-01-01T00:00:00+00:00', '--job-id', '1393', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmph6rgv2h9']
[2024-12-19T08:54:53.944+0000] {standard_task_runner.py:105} INFO - Job 1393: Subtask first_task
[2024-12-19T08:54:54.067+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v2.first_task scheduled__2023-01-01T00:00:00+00:00 [running]> on host 8d142e9c83a7
[2024-12-19T08:54:54.221+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mateo' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2023-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-01-01T00:00:00+00:00'
[2024-12-19T08:54:54.226+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-19T08:54:54.279+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-12-19T08:54:54.284+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task']
[2024-12-19T08:54:54.308+0000] {subprocess.py:99} INFO - Output:
[2024-12-19T08:54:54.312+0000] {subprocess.py:106} INFO - hello world, this is the first task
[2024-12-19T08:54:54.315+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-12-19T08:54:54.351+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-19T08:54:54.355+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=first_task, run_id=scheduled__2023-01-01T00:00:00+00:00, execution_date=20230101T000000, start_date=20241219T085453, end_date=20241219T085454
[2024-12-19T08:54:54.403+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-19T08:54:54.430+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-19T08:54:54.436+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
